frameworks/cassandra/tests/test_udf.py

```diff
@@ -0,0 +1,52 @@
+from typing import Any, Dict, Iterator, List
+import logging
+import pytest
+import sdk_jobs
+import sdk_install
+from tests import config
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module", autouse=True)
+def configure_package(configure_security: None) -> Iterator[None]:
+    test_jobs: List[Dict[str, Any]] = []
+    try:
+        test_jobs = config.get_udf_jobs()
+        # destroy/reinstall any prior leftover jobs, so that they don't touch the newly installed service:
+        for job in test_jobs:
+            sdk_jobs.install_job(job)
+
+        service_options = {
+            "service": {
+                "name": config.SERVICE_NAME,
+                "security": {"authentication": {"enabled": True}, "authorization": {"enabled": True}},
```

Comment:
I'm surprised this works... your other PR where you add these configs hasn't been merged yet

Reply:
correct, its just do not apply options which are not available rather then failing. but removed it as well.